### ISSUE TITLE
Make reporter option require outputFile option

### ISSUE THREAD

**Issue by Tapchicoma** [shyiko/ktlint#71]
I've tried to introduce new reporter option in the https://github.com/JLLeitschuh/ktlint-gradle plugin and one problem so far - now console output lacks old plain errors when reporter is set to `checkstyle` or `json`.

I'd like to have old behaviour for ktlint run output, but introduce new option `--outputFile` to where reporter will write it output.

**Comment by shyiko** [comment]
@Tapchicoma sounds reasonable. 

Instead of introducing another CLI option, how about something like
`ktlint --reporter=plain?group_by_file --reporter="checkstyle > target/checkstyle.xml" --reporter="html > target/ktlint.html"`?

**Comment by Tapchicoma** [comment]
Hm. If you don't want to introduce another option then I'd like to have more standard approach for additional command line option arguments:

```
ktlint --reporter plain?group_by_file --reporter checkstyle=target/checkstyle.xml
```

or 

```
ktlint --reporter plain?group_by_file --reporter checkstyle,output=target/checkstyle.xml
```

See [getsubopt() documentation](http://pubs.opengroup.org/onlinepubs/9699919799/functions/getsubopt.html) for reference.

**Comment by shyiko** [comment]
@Tapchicoma it's not that I don't want to introduce another option, it just `--outputFile` can potentially  make things complicated for multi-format reporting (checkstyle and html, for example). 
Anyway, I like your 2nd example (`ktlint --reporter plain?group_by_file --reporter checkstyle,output=target/checkstyle.xml`). I think this is the way to go. Also, thank you for the link :bowing_man:.

**Comment by crossonrose** [comment]
They will all die 10

**Comment by shyiko** [comment]
Implemented in [0.10.0](https://github.com/shyiko/ktlint/releases/tag/0.10.0).
